test/engine-luatest/gh_8609_default_field_value_func_test.lua

```diff
@@ -0,0 +1,226 @@
+local t = require('luatest')
+local g = t.group('gh-8609', {{engine = 'memtx'}, {engine = 'vinyl'}})
+
+g.before_all(function(cg)
+    local server = require('luatest.server')
+    cg.server = server:new({alias = 'master'})
```

Comment:
Nit: `alias` may be omitted: `cg.server = server:new()`.

Reply:
Removed.